src/cli/peakdetector/peakdetectorcli.cpp

```diff
@@ -1,13 +1,24 @@
 #include "common/downloadmanager.h"
 #include "Compound.h"
 #include "csvparser.h"
+#include "common/logger.h"
 #include "mavenparameters.h"
 #include "masscutofftype.h"
 #include "mzUtils.h"
 #include "peakdetectorcli.h"
 
 PeakDetectorCLI::PeakDetectorCLI()
 {
+    QString parentFolder = "ElMaven";
+    QString logFile = "peakdetector_cli.log";
```

Comment:
The same `logFile` name is defined in main as well. Use that so it does not have to be maintained in both the files.

Reply:
Made the change.